src/Guide/Config.hs

```diff
@@ -60,6 +71,11 @@ instance FromJSON Config where
     _adminPassword <- o .:? "admin-password" .!= _adminPassword def
     _prerender     <- o .:? "prerender"      .!= _prerender def
     _discussLink   <- o .:? "discuss-link"   .!= _discussLink def
+    _portMain      <- o .:? "portMain"       .!= _portMain def
```

Comment:
The rest of fields in this instance use snake-case, these three fields should use snake-case as well.